packages/terra-modal-manager/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+  * Changed
+    * Added Accessibility support for nested ModalManager disclosures.
```

Comment:
Can we update this one as well? (Same as https://github.com/cerner/terra-framework/pull/1652#discussion_r1182735947)
Other than that looks good to me.